bookmark.html

```diff
@@ -14,7 +14,9 @@ <h1>Bookmarks</h1>
     </header>
     <main>
       <article data-js="card">
-        <button class="material-symbols-outlined">bookmark</button>
+        <button data-js="bookmark" class="material-symbols-outlined">
```

Comment:
I think you can use BEM naming here as well
```suggestion
        <button data-js="bookmark" class="card__bookmark">
```

Reply:
Unfortunately it seems to be impossible to change the class name "material-symbols-outlined" to anything else without losing the icon font.